schedule/sles4sap/qam/common/ha_supportserver.yaml

```diff
@@ -0,0 +1,34 @@
+---
+name: ha_supportserver
```

Comment:
In all the HA QAM tests, I've prefixed both files and names with `qam_` like:
`name:           qam_ha_qdevice_node` and `schedule/ha/qam/common/qam_ha_qdevice_node.yaml` 
My solution is perhaps not the better but we should agree on a common naming convention.

Reply:
Ok, I will prefix all of them.